react/ThumbnailItem.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/** 
+ * Thumbnail Item Component.
+ * Display a thumbnail image with an on click event well defined. 
+ */
+class ThumbnailItem extends Component {
+  
+  /** 
+   * Function that is called when the thumbnail image is clicked 
+   */
+  handleClick = (event) => {
+    event.preventDefault()
+    const { image, onClick } = this.props
+    onClick(image)
+  }
+
+  render() {
+    const { imageUrl, imageText } = this.props.image
+
+    return (
+      <a href="#" onClick={this.handleClick}>
```

Comment:
Not sure about using an `a` tag if there's no link to be followed. Wouldn't a div work similarly?

Reply:
I was trying to use the on mouse hover behavior that the <a> tag have by default. But I realized that it is possible to use a <div> tag with some tachyons classes which provides it. 
Fixed!